components/FormStyles/RadioButton.tsx

```diff
@@ -1,28 +1,26 @@
 import styled from 'styled-components'
+import { COLORS } from '../../constants/colors'
 
 export const RadioButton = styled.input`
```

Comment:
Can you add a `:hover` rule to this as well bc for some reason when you hover over the button an ugly background appears:
<img width="169" alt="Screen Shot 2022-04-20 at 9 03 56 PM" src="https://user-images.githubusercontent.com/67511512/164351130-25bca1c7-f148-41e2-87cb-3203b02d9f75.png">